app/js/components/LabResultsList.jsx

```diff
@@ -14,13 +14,19 @@ const patientUUID = process.env.NODE_ENV !== 'production'
   ? 'b2231edd-f62b-47fc-a9c7-feb49c63721c' // your patient uuid will go here
   : '76f0fd80-2b5b-496a-8b68-539d7e532ad2';
 
-const Cell = ({ value, columnName, type }) => {
+const Cell = ({
+  value, columnName, type, navigate,
+}) => {
   if (type === 'single') {
     const hasNoEncounter = value.status === 'Ordered';
     const isPanel = value.order.concept.set;
     if (columnName === 'TYPE') {
       return (
-        <div className="table_cell type">
+        <div
+          className="table_cell type" onClick={(e) => {
+            event.preventDefault();
```

Comment:
`event` won't be defined since it's passed in as `e` but instead would fall back to window.event